inspirehep/modules/search/query_suggestions.py

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Shouldn't this file be split between signals and receivers? Ideally signals could be moved into `invenio-query-parser` then.